src/FAudio_platform_win32.c

```diff
@@ -474,7 +476,7 @@ void FAudio_PlatformInit(
 	args->events[0] = audioEvent;
 	args->events[1] = data->stopEvent;
 	args->audio = audio;
-	if (flags & FAUDIO_1024_QUANTUM) args->updateSize = args->format.Format.nSamplesPerSec / (1000.0 / (64.0 / 3.0));
+	if (flags & FAUDIO_1024_QUANTUM) args->updateSize = (UINT)ceil(args->format.Format.nSamplesPerSec / (1000.0 / (64.0 / 3.0)));
```

Comment:
This changes behavior, if it should be ceil'd we can do this in another commit

Reply:
I just thought it makes sense to use `ceil()` for a floating point calculation yielding a buffer size, but it probably does not matter.